lib/flowmailer_rails/mailer.rb

```diff
@@ -52,6 +52,15 @@ def submit_message(json)
       response
     end
 
+    def handle_failure(response)
+      raise ExpiredAccessTokenError if response.status == 401
+
+      too_big_error = response.body["allErrors"]&.find { |error| error["code"] == "message.toobig" }
```

Comment:
Should we check if `response.body` has any value and/or is a hash before we try to extract `allErrors`?

Reply:
Yeah that's true, body could be a string. 
But I was writing tests now for the cases of `body: nil` and `body: "string"` and for some reason, that escapes me completely, it works and raises a `DeliveryError` as it should without me changing the code. Maybe there's some stubbing magic in action that I'm unaware of. 🤔

Reply:
`[]` should work for strings though.